ptb-lm.py

```diff
@@ -413,7 +421,21 @@ def run_epoch(model, data, is_train=False, lr=1.0):
         iters += model.seq_len
         if args.debug:
             print(step, loss)
-        if is_train:  # Only update parameters if training 
+
+        # todo computing gradient norms
+        if args.compute_gradient:
```

Comment:
and on the validation we iterate over the hidden states and get their gradients, turn them to norms